utils/shared.js

```diff
@@ -672,6 +672,21 @@ const cleanSurveyData = (data) => {
     return data;
 }
 
+const sop = async (req, res) => {
```

Comment:
please name something more descriptive.. SOP stands for "standard operating procedure" which could be applied to many things

Reply:
Solved